ci/fireci/fireci/commands.py

```diff
@@ -61,3 +63,38 @@ def smoke_tests(app_build_variant, test_apps_dir):
             os.path.join(cwd, 'build', 'm2repository')),
         workdir=location,
     )
+
+
+@click.argument('issue_number', required=True)
+@click.argument('repo_name', required=True)
+@click.argument('auth_token', required=True)
+@ci_command()
+def api_information(auth_token, repo_name, issue_number):
+  """Comments the api information on the pr"""
+
+  gradle.run('apiInformation')
+  gradle.run('generateApiTxtFile')
```

Comment:
you can invoke them in one gradle invocation, e.g. `gradle.run("task1", "task2")`.

Also you have 3 java tasks defined above, but here you're invoking only 2, is one of them not needed?

Reply:
I have only 2 tasks defined above?

Reply:
I can see 3 :)